web/wp-content/themes/dctx-v2/template-parts/content-buddypress.php

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-f8wqfn.php, but found phpcs-scan-F8wqFn.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).